meta/applications/yc_s15.md

```diff
@@ -0,0 +1,466 @@
+sponsor).
+
+When we talked with a former employee of CodeNow, they said that the process of
+forming a partnership with a school and reaching its students took 2-3 months
+of full-time work. Our model allows us to avoid all of that work and skip the
+red-tape and bureaucracy of partnering with schools. This is similar to
+Airbnb's model of not having hosts on staff or owning property, allowing us to
+scale in-person coding education which is something we haven't seen done
+before.
+
+**Have you incorporated, or formed any legal entity (like an LLC) yet?**
+
+Yes.
+
+**What kind of entity and in what state or country was the entity formed?**
+
+hackEDU is incorporated in the state of Delaware as a non-stock non-profit
+corporation.
+
+**If you're already incorporated, when were you? If you've had funding, how
+much and from whom?**
+
+HackEDU was incorporated in January of 2014. We've secured a small amount
+($10k) of funding from various corporate sponsors.
+
+**Have you obtained your 501(c)(3) tax-exempt status yet?**
+
+We have not, but are in the final stages of closing a fiscal sponsor and are in
+the application process.
+
+**Please provide any other relevant information about the structure or
+formation of the company.**
+
+N/A
+
+**Are any of the following true?**
+
+* You are the only founder.
+* You are a student who may return to school when the next term starts.
+* Half or more of your group can't move to the Bay Area.
+* One or more founders will keep their current jobs.
+
+* [ ] Yes
+* [x] No
+
+**If you had any other ideas you considered applying with, please list them.
+One may be something we've been waiting for.**
+
+**Please tell us something surprising or amusing that one of you has
+discovered. (The answer need not be related to your project.)**
+
+`fortune | cowsay | lolcat` speaks volumes of truth.
+
+**What convinced you to apply to Y Combinator?**
+
+Engaging with the community on Hacker News and meeting several YC founders.
+
+**How did you hear about Y Combinator?**
+
+Hacker News.
+
+### Founder Profiles
+
+#### Zach Latta
+
+**Name**
+
+Zach Latta
+
+**Email Address**
+
+zach@hackedu.us
+
+**Age**
+
+17
+
+**Gender**
+
+Male
+
+**Schools, Degrees and Years of Graduation**
+
+* El Segundo High School, 2014
+* West Los Angeles College, N/A
+
+**Employers, Position / Titles and Dates**
+
+* Yo, Lead Backend Engineer, July 2014 - September 2014
+* Grasp, Technical Lead, June 2013 - January 2014
+* Run Games, Junior Programmer, February 2013 - February 2014
+
+**Personal website**
+
+http://zachlatta.com
+
+**Github URL**
+
+https://github.com/zachlatta
+
+**Linkedin URL**
+
+https://www.linkedin.com/in/zachlatta
+
+**Facebook URL**
+
+https://facebook.com/crynix
+
+**Twitter URL**
+
+https://twitter.com/zachlatta
+
+**Please tell us about the time you most successfully hacked some
+(non-computer) system to your advantage.**
+
+I hacked my high school education. I exploited a loophole in the California
```

Comment:
I don't know if this is the best example.  Something seems contradictory given the rest of you app..